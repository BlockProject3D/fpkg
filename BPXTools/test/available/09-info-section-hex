export NAME="Info (Section + Hex)"
export COMMAND="-f test.bpx info -d 0 -x"
export DESC="Test the info command"
export STATUS=0