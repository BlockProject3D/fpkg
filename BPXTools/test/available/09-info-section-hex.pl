$Test = {
    Name => "Info (Section + Hex)",
    Command => "-f test/available/test.bpx info -d 0 -x",
    Description => "Test the info command",
    Status => 0
};

sub TestBegin {

}

sub TestEnd {
    return 1;
}
