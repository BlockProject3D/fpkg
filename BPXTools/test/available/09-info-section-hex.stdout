====> BPX Main Header <====
Type: P
Version: 1
File size: 1629
Number of sections: 2
====> End <====

4C 49 43 45 4E 53 45 2E 74 78 74 00 
